local_units/serializers.py

```diff
@@ -246,12 +246,15 @@ class PrivateLocalUnitDetailSerializer(NestedCreateMixin, NestedUpdateMixin):
     class Meta:
         model = LocalUnit
         fields = (
+            "id",
             "local_branch_name",
             "english_branch_name",
             "type",
             "country",
             "created_at",
+            "created_by",
```

Comment:
Is this required?

Reply:
@thenav56 I added this to check `self.assertEqual(response_updated_1["modified_by"], response["created_by"])`